ops/terraform/modules/resources/s3/main.tf

```diff
@@ -45,6 +45,8 @@ resource "aws_s3_bucket" "main" {
 resource "aws_s3_bucket_public_access_block" "main" {
   bucket = aws_s3_bucket.main.id
 
-  block_public_acls   = true
-  block_public_policy = true
+  ignore_public_acls      = true
```

Comment:
keith, can you define what 'public' means? will s3 access be constrained to operating in vpn? mainly just curious how this will affect some stuff we devs do; for example:
`aws s3 cp s3://bfd-prod-etl-577373831711/Done/2022-02-25T12:00:00Z/bene_upd20220305065502.txt - | grep -E '^(UPDATE|INSERT)\|<some bene_id>\|'`